pkg/costattribution/active_tracker.go

```diff
@@ -136,17 +187,21 @@ func (at *ActiveSeriesTracker) Decrement(lbls labels.Labels) {
 	defer bufferPool.Put(buf)
 	at.fillKeyFromLabels(lbls, buf)
 	at.observedMtx.RLock()
-	as, ok := at.observed[string(buf.Bytes())]
+	c, ok := at.observed[string(buf.Bytes())]
 	if ok {
-		nv := as.Dec()
+		nv := c.activeSeries.Dec()
+		if nativeHistogramBucketNum >= 0 && c.nativeHistograms.Load() > 0 {
```

Comment:
we don't check for consistency of c.activeSeries so I think we should do this:
```suggestion
		if nativeHistogramBucketNum >= 0  {
```